base/applications/calc/lang/uk-UA.rc

```diff
@@ -1,8 +1,7 @@
 /*
- * PROJECT:     ReactOS Calculator
+ * PROJECT:     ReactOS Calc
```

Comment:
Is this change synchronised with the English translation or the calculator.c files?

Reply:
It was used in one of the stasm tweaked headers which do reflect the 0.4.15-dev-style already.